contracts/token-manager/TokenManager.sol

```diff
@@ -82,164 +78,53 @@ abstract contract TokenManager is ITokenManager, ITokenManagerType, Operatable,
         revert NotSupported();
     }
 
+    /**
+     * @notice Returns implementation type of this token manager.
+     * @return uint256 The implementation type of this token manager.
+     */
+    function implementationType() external pure returns (uint256) {
+        revert NotSupported();
+    }
+
     /**
      * @notice A function that should return the token address from the setup params.
-     * @param params The setup parameters.
+     * @param params_ The setup parameters.
      * @return tokenAddress_ The token address.
      */
-    function getTokenAddressFromParams(bytes calldata params) external pure returns (address tokenAddress_) {
-        (, tokenAddress_) = abi.decode(params, (bytes, address));
+    function getTokenAddressFromParams(bytes calldata params_) external pure returns (address tokenAddress_) {
+        (, tokenAddress_) = abi.decode(params_, (bytes, address));
     }
 
     /**
      * @notice Setup function for the TokenManager.
      * @dev This function should only be called by the proxy, and only once from the proxy constructor.
      * The exact format of params depends on the type of TokenManager used but the first 32 bytes are reserved
      * for the address of the operator, stored as bytes (to be compatible with non-EVM chains)
-     * @param params The parameters to be used to initialize the TokenManager.
+     * @param params_ The parameters to be used to initialize the TokenManager.
      */
-    function setup(bytes calldata params) external override(Implementation, IImplementation) onlyProxy {
-        bytes memory operatorBytes = abi.decode(params, (bytes));
+    function setup(bytes calldata params_) external override(Implementation, IImplementation) onlyProxy {
+        bytes memory operatorBytes = abi.decode(params_, (bytes));
+
+        // slither-disable-next-line uninitialized-local
+        address operator;
```

Comment:
I'd keep the init for readability if gas cost isn't significant, and since this is incurred in deployment.